.gitignore

```diff
@@ -1,2 +1,3 @@
 *.iml
 target/
+.idea/
```

Comment:
This change is irrelevant, let's revert it.

Reply:
I have changed the PR title and reverted the change to .gitignore